cloudinit/net/ifconfig.py

```diff
@@ -0,0 +1,13 @@
+# This file is part of cloud-init. See LICENSE file for license information.
```

Comment:
Since I'm new to this, I just kept the same boilerplate as I thought it might be needed and would already be there if more things were needed.  This was a quick thing I did, just to get it working.  I can look into reducing the boilerplate though.

Reply:
Thanks. If the ifconfig definition were more complicated I'd agree with this choice, but since it's just one short function no need to add the extra file. Just move the contents of the `available()` function here into the `IfConfigActivator.available()` and remove this file and we should be good. 

> Since I'm new to this

No worries :) this is looking like a great early contribution.